frontstage/views/secure_messaging/create_message.py

```diff
@@ -16,32 +16,35 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-@secure_message_bp.route('/create-message', methods=['GET', 'POST'])
+@secure_message_bp.route('/create-message/', methods=['GET', 'POST'])
 @jwt_authorization(request)
 def create_message(session):
+    case_id = request.args.get('case_id')
+    survey = request.args.get('survey')
+    ru_ref = request.args.get('ru_ref')
```

Comment:
Should there be some sort of verification that these aren't empty?
If a use navigates to the `/create-messages URL` without the query strings, the page will currently still try to send the message with these fields missing.

Reply:
In the current implementation they should not be able to send a message apart from through the survey list

Reply:
But yes atm this should not work without these so should have validation